antarest/storage/service.py

```diff
@@ -306,6 +319,52 @@ def edit_study(
         )
         return updated
 
+    def change_owner(
+        self, study_id: str, owner_id: int, params: RequestParameters
+    ) -> None:
+        study = self._get_study(study_id)
+        self._assert_permission(
+            params.user, study, StudyPermissionType.MANAGE_PERMISSIONS
+        )
+        new_owner = self.user_service.get_user(owner_id, params)
+        study.owner = new_owner
+        self.repository.save(study)
+
+    def add_group(
+        self, study_id: str, group_id: str, params: RequestParameters
+    ) -> None:
+        study = self._get_study(study_id)
+        self._assert_permission(
+            params.user, study, StudyPermissionType.MANAGE_PERMISSIONS
+        )
+        group = self.user_service.get_group(group_id, params)
```

Comment:
This line create intermodule dependency. We could avoid by:
- if admin, we authorize any group without verify group exists...
- if group admin or owner, user can only add group where he is admin according to his jwt. 

Since we remain monolithic this line can remain too